cli/flox-rust-sdk/Cargo.toml

```diff
@@ -32,6 +33,7 @@ tracing.workspace = true
 url.workspace = true
 uuid.workspace = true
 walkdir.workspace = true
+futures.workspace = true
```

Comment:
In the English alphabet `f` comes well before `w` :)

Reply:
TIL ;)